CRM/Utils/Hook.php

```diff
@@ -1805,6 +1805,28 @@ public static function triggerInfo(&$info, $tableName = NULL) {
     );
   }
 
+  /**
+   * Define the list of fields supported in APIv4 data-translation.
+   *
+   * @param array $fields
+   *   List of data fields to translate, organized by table and column.
+   *   Omitted/unlisted fields are not translated. Any listed field may be translated.
+   *   Values should be TRUE-ish.
```

Comment:
I find this confusing - the value should be TRUE - but you are WARNING that setting a different value might resolve to true - it comes across as you saying 'there is no recommendation  - do what you want' and also is a lot more cognitive load than just daying 'the value should be boolean' and expecting people to have enough sense/coding skills to realise that they should set a boolean or something weird might happen

Reply:
It was because the original list of translatable fields (`SchemaStructure`) had TRUE-ish things rather than strict booleans. But since we're not feeding off that now, we can document it as straight-up boolean.